tests/unit/conftest.py

```diff
@@ -124,19 +126,30 @@ def db_name() -> str:
 
 @pytest.fixture
 def account_id_url(settings: Settings) -> str:
-    return f"https://{settings.server}{ACCOUNT_URL}"
+    if settings.account_name is None:
```

Comment:
Could account_name ever be None in our tests?
I think we require ACCOUNT_NAME env variable to be provided.
So I think we can leave only second branch

Reply:
It can be `None`. It's actually set to `None` on l. 98 of `conftest.py` as a default and fails all over the place when I remove the first branch.

Reply:
I also changed this logic to deal with the possibility of `settings.account+name` being an empty string.